src/pages/Individual/Individual.js

```diff
@@ -1,41 +1,120 @@
 
 import IndividualItem from './IndividualItem/IndividualItem';
 import IndividualFilter from './IndividualFilter/IndividualFilter';
+import EmptyIndividual from './EmptyIndividual/EmptyIndividual';
+import StyledCheckBox from '../../components/StyledCheckBox/StyledCheckBox';
+import InvestmentBtn from './InvestmentBtn/InvestmentBtn';
+
+import { BASE_URL, INDIVIDUAL } from '../../config';
+import {
+  GRADE_BAR_RANGE,
+  EARNING_BAR_RANGE,
+  GRADE_CONVERSION,
+} from './IndividualFilter/filterRange';
 
 function Individual() {
-  const [wholeCheck, setWholeCheck] = useState(false);
+  const [individual, setIndividual] = useState(null);
+  const [copyIndividual, setCopyIndividual] = useState([]);
+  const [gradeFilterRange, setGradeFilterRange] = useState({
+    start: 0,
+    end: GRADE_BAR_RANGE.length - 1,
+  });
+  const [earningFilterRange, setEarningFilterRange] = useState({
+    start: 0,
+    end: EARNING_BAR_RANGE.length - 1,
+  });
+
   const [isFilterOn, setIsFilterOn] = useState(false);
+  const [isInvestBtnOn, setIsInvestBtnOn] = useState(false);
+
+  const [checkedItem, setCheckedItem] = useState([]);
+  const [isPurposeChecked, setIsPurposeChecked] = useState(false);
+
+  const isInitialMount = useRef(true);
 
-  const handleCheckBox = e => {
-    setWholeCheck(!wholeCheck);
+  useEffect(() => {
+    fetch(`${BASE_URL}${INDIVIDUAL}`)
+      .then(res => res.json())
+      .then(res => {
+        setIndividual(res.results);
+        setCopyIndividual(res.results);
+      });
+  }, []);
+
+  const handleFilter = (grade, earningPercent, inputValue, direction) => {
+    if (grade) {
+      if (direction === 'start') {
+        setGradeFilterRange(prev => ({ start: inputValue, end: prev.end }));
+      } else if (direction === 'end') {
+        setGradeFilterRange(prev => ({ start: prev.start, end: inputValue }));
+      }
+    }
+
+    if (earningPercent) {
+      if (direction === 'start') {
+        setEarningFilterRange(prev => ({ start: inputValue, end: prev.end }));
+      } else if (direction === 'end') {
+        setEarningFilterRange(prev => ({ start: prev.start, end: inputValue }));
+      }
+    }
   };
 
+  useEffect(() => {
+    if (isInitialMount.current) {
+      isInitialMount.current = false;
+    } else {
+      const filteredList = individual.filter(item => {
+        return (
+          item.earningRate >= EARNING_BAR_RANGE[earningFilterRange.start] &&
+          item.earningRate <= EARNING_BAR_RANGE[earningFilterRange.end] &&
+          GRADE_CONVERSION[item.grade] >= gradeFilterRange.start &&
+          GRADE_CONVERSION[item.grade] <= gradeFilterRange.end &&
+          ((isPurposeChecked && item.title === '대환대출') || !isPurposeChecked)
+        );
+      });
+      setCopyIndividual(filteredList);
+    }
+  }, [earningFilterRange, gradeFilterRange, isPurposeChecked]);
+
+  useEffect(() => {
+    const hasCheckedItem = checkedItem.length !== 0;
+    setIsInvestBtnOn(hasCheckedItem);
```

Comment:
```suggestion
    setIsInvestBtnOn(!!checkedItem.length);
```
0은 falsy값 입니다!
falsy의 반대는 true 입니다. -> !falsy
true의 반대는 false입니다. -> 0
위와 같이 표현 가능합니다!